src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -351,6 +458,14 @@ const INPUTS = [
     type: ITEM_TYPES.INFO_TEXT,
     title: texts.consul.startNew.proposalVideoUrlInfo
   },
+  {
+    buttonTitle: texts.consul.startNew.newProposalImageAddButtonTitle,
+    infoText: texts.consul.startNew.newProposalImageAddInfoText,
+    label: texts.consul.startNew.newProposalImageAddTitle,
+    name: 'image',
+    rules: { required: false },
+    type: ITEM_TYPES.PICKER
```

Comment:
`type` is always the first key in this structure but now its alphabetical - just for the new entry. i think it helped to have `type` first to quickly differentiate between the entries. what do you think could be the best structure to keep the json easily readable?

Reply:
You say right. In terms of readability, it would be great if they were all in the same structure. I will fix this.

Reply:
8130952b6dccca99da959e64754c14698f3b10bf